RecoEcal/EgammaCoreTools/interface/EcalClusterLazyTools.h

```diff
@@ -33,7 +33,9 @@
 #include "CalibCalorimetry/EcalLaserCorrection/interface/EcalLaserDbRecord.h"
 #include "RecoLocalCalo/EcalRecAlgos/interface/EcalSeverityLevelAlgoRcd.h"
 #include "FWCore/Framework/interface/ConsumesCollector.h"
-
+#include "CondFormats/EcalObjects/interface/EcalPFRecHitThresholds.h"
+#include "CondFormats/DataRecord/interface/EcalPFRecHitThresholdsRcd.h"
+#include "RecoEgamma/EgammaTools/interface/EgammaLocalCovParamDefaults.h"
```

Comment:
hi @swagata87 @cms-sw/reconstruction-l2 - it looks like this change creates a circular dependency between RecoEgamma/EgammaTools and RecoEcal/EgammaCoreTools. I'd suggest deciding which should be lower level than the other and moving things appropriately.  Thanks!

Reply:
Hi,
sorry about this  :-(
indeed RecoEcal should not include any file from RecoEgamma, but the opposite is okay.
so, to fix this, I can move the file EgammaLocalCovParamDefaults.h to /RecoEcal/EgammaCoreTools/interface/ , that would solve the circular dependency.
thanks for pointing this out.